.github/workflows/release.yml

```diff
@@ -0,0 +1,36 @@
+name: Release PECL Preparation
+
+on:
+  release:
+    types: [published]
+  workflow_dispatch:
+
+jobs:
+  package:
+    name: Prepare package
+    runs-on: macos-latest
+
+    steps:
+    - name: Setup PHP
+      uses: shivammathur/setup-php@v2
+      with:
+        php-version: '8.0'
+
+    - name: Checkout Source
+      uses: actions/checkout@v3
+      with:
+          submodules: recursive
+          fetch-depth: 0
+
+    - name: Get release tag
+      uses: awslabs/aws-crt-builder/.github/actions/release-tag@main
+      id: tag
+
+    - name: Prepare Package Script
+      run: dev-scripts/prepare_release.sh --name aws-crt --user aws-crt --email aws-sdk-common-runtime@amazon.com --version ${{ steps.tag.outputs.release_tag }} --notes NOTES
```

Comment:
what is `NOTES`?

if this is a TODO thing, please add a TODO, or make the `--notes` flag optional